examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) Granulate. All rights reserved.
+ * Licensed under the AGPL3 License. See LICENSE.txt for license information.
+ */
+
+#include "PyPerfNativeStackTrace.h"
+
+#include <sys/uio.h>
+#include <unistd.h>
+
+#include <cstdio>
+#include <cstring>
+
+#include "PyPerfLoggingHelper.h"
+
+namespace ebpf {
+namespace pyperf {
+
+const uint8_t *NativeStackTrace::stack = NULL;
```

Comment:
No nicer way w/o the globals? Perhaps libunwind allows you to pass a "context" parameter (so you can pass the `NativeStackTrace` object?)

Reply:
There is a context, but it's already used by UPT with an internal struct that isn't visible to the user (the return value of `_UPT_create`). I can use my own context that will hold the pointer to this struct and wrap all the other accessors with functions that passes it to the UPT accessors.

Reply:
I see. Okay. Can drop a comment on them maybe.

Reply:
?

Reply:
Added